infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/provisioner/volume/VolumeNameGenerator.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.docker.provisioner.volume;
+
+/**
+ * Helps to generate volume names and matches existing volumes to workspace.
+ *
+ * @author Alexander Garagatyi
+ */
+public class VolumeNameGenerator {
```

Comment:
I'd propose to rework this class in the following way:
```java
...
public class VolumeNames {

  public static String generate(String workspaceId, String originVolumeName)...

  public static boolean matches(String volumeName, String workspaceId)...

  public static boolean matches(String volumeName, String originVolumeName, String workspaceId)...
  
  ....
}
```

Reply:
+1

Reply:
done